core/electron/package.json

```diff
@@ -6,9 +6,22 @@
   "engines": {
     "node": "^18.0.0 || ^20.0.0"
   },
+  "exports": {
+    "./ElectronBackend": {
+      "types": "./lib/esm/ElectronBackend.d.ts",
+      "import": "./lib/esm/ElectronBackend.js",
+      "require": "./lib/cjs/ElectronBackend.js"
+    },
+    "./ElectronFrontend": {
+      "types": "./lib/esm/ElectronFrontend.d.ts",
+      "import": "./lib/esm/ElectronFrontend.js",
+      "require": "./lib/cjs/ElectronFrontend.js"
+    }
+  },
   "scripts": {
-    "build": "npm run -s build:cjs && npm run -s webpack:test",
+    "build": "npm run -s build:cjs && npm run -s build:esm && npm run -s webpack:test",
     "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module esnext --outDir lib/esm",
```

Comment:
Why deliver both CJS and ESM ? Wouldn't it be less painful to just switch to ESM on 5.0 ?

Reply:
yeah, I was initially thinking this would go into one of the 4.X versions, but I'm now more certain this should be in 5.x  - as I go through with modifying DTA, I'm noticing pain points with delivering dual cjs/esm, both in these packages and our electron-authorization pkg. When running an ESM module, and using named imports (like "@itwin/electron-authorization/Main"), electron complains at run time that it doesn't like electron-auth being CJS. And seems like the only way to solve that is to set 'type': "module".